.github/workflows/bintray.yml

```diff
@@ -0,0 +1,19 @@
+name: Upload Android AAR to bintray
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: "Upload AAR to bintray"
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Upload
+      run: cd ffi/java/android/project/mcl-android; ../gradlew bintrayUpload -PbintrayUser=${{secrets.BINTRAY_USER}} -PbintrayApikey=${{secrets.BINTRAY_APIKEY}}
```

Comment:
when creating the PR back to herumi/mcl, we should describe in the PR description that does secrets need to be configured in Github and a bintray account needs to be setup.